packages/plugin-dev/src/browser/hosted-plugin-preferences.ts

```diff
@@ -30,13 +30,24 @@ export const HostedPluginConfigSchema: PreferenceSchema = {
             description: 'Using inspect or inspect-brk for Node.js debug',
             default: 'inspect',
             enum: ['inspect', 'inspect-brk']
+        },
+        'hosted-plugin.launchOutFiles': {
+            type: 'array',
+            items: {
+                type: 'string'
+            },
+            description: 'Array of glob patterns for locating generated JavaScript files (`${pluginPath}` will be replaced by plugin actual path).',
+            default: [
+                '${pluginPath}\\out\\**\\*.js'
```

Comment:
Shouldn't we use forward slashes here?

Reply:
Before this change - this is what `outFiles` resulted in:
![image](https://user-images.githubusercontent.com/71069170/119254798-1c21ee00-bbc1-11eb-968e-d03fb27e3cb6.png)

So `outfiles` is file system path - therefore I don't see need to continue with forward slashes as before. WDYT?

Reply:
Backslashes are Windows-specific. So no matter what you do, it shouldn't be hardcoded to one or the other.

Reply:
@marechal-p 

Added forward slashes. Tried also doing new Path().join but it adds '/' also.  Saw that we already used forward slashes also in other preferences, i.e.:
![image](https://user-images.githubusercontent.com/71069170/119507568-9e541300-bd77-11eb-93ca-1c7ada3c23a1.png)

Reply:
You are right to look at other configs. Most of the time forward slashes are favored, and converted to backslashes for Windows when required. Thanks for the update!